docs/reference-architectures/data/monitoring.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Solution Accelerator for Monitoring Azure Databricks
+titleSuffix: Azure Solution Accelerators
+description: A scala library to deploy to a Databricks cluster to enable monitoring of metrics and logging data in Azure Log Analytics
+author: petertaylor9999
+ms.date: 01/28/2019
+ms.topic:
+ms.service:
+ms.subservice:
+---
+
+# Solution Accelerator: Monitoring Azure Databricks using Azure Log Analytics
```

Comment:
Is "Solution accelerator" the official name for this? I'm wondering this term means something to customers. I'd think "package" or "library" would resonate better. Also I'm a little doubtful about including it in the title.